app/src/main/res/values/strings.xml

```diff
@@ -8,4 +8,20 @@
     <string name="action_zoom_out">Zoom out</string>
     <string name="action_zoom_in">Zoom in</string>
     <string name="action_jump_to_page">Jump to page</string>
+    <string name="action_view_document_properties">Document Properties</string>
```

Comment:
For consistency, these should be "Document properties", etc. rather than title case.